server/plugin/virtual_agent.go

```diff
@@ -329,13 +328,18 @@ func (p *Plugin) ProcessResponse(data []byte) error {
 				}
 			}
 		case *OutputImage:
-			var post *model.Post
-			post, err = p.CreateOutputImagePost(res, userID)
-			if err != nil {
-				return err
+			linkContents := strings.Split(res.Value, "/")
+			if len(linkContents) < 1 {
+				if _, err = p.DM(userID, fmt.Sprintf("Image: %s", res.AltText)); err != nil {
+					return err
+				}
+
+				p.API.LogError(InvalidImageLinkError, "Link", res.Value)
+				return errors.New(InvalidImageLinkError)
 			}
 
-			if _, err = p.dm(userID, post); err != nil {
+			completeFileName := linkContents[len(linkContents)-1]
+			if _, err = p.DM(userID, fmt.Sprintf("![%s](%s)", completeFileName, res.Value)); err != nil {
```

Comment:
If we are already receiving the `res.AltText` in the response, then can we not use the same instead of parsing the logic to get the completeFileName?

Reply:
@srkgupta That Alt text field is optional. So, there will be cases when the field will be empty.

Reply:
Got it. Thanks for the clarification @raghavaggarwal2308